protected/humhub/modules/content/widgets/WallCreateContentMenu.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2022 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\content\widgets;
+
+use humhub\modules\content\components\ContentActiveRecord;
+use humhub\modules\content\components\ContentContainerActiveRecord;
+use humhub\modules\content\widgets\stream\WallStreamEntryWidget;
+use humhub\modules\ui\menu\MenuLink;
+use humhub\modules\ui\menu\widgets\Menu;
+use Yii;
+
+/**
+ * WallCreateContentMenu is the widget for Menu above wall create content Form
+ *
+ * @property-read ContentContainerActiveRecord $contentContainer
+ * @author luke
+ * @since 1.13.0
```

Comment:
Small note: Please use `1.13` and not include the patch level.

Reply:
Commit https://github.com/humhub/humhub/pull/5827/commits/ee09a612696597d5d6b2fd83c78d188941ce5f2c